supervisor/store/git.py

```diff
@@ -53,7 +53,7 @@ async def load(self):
                 git.GitCommandError,
             ) as err:
                 _LOGGER.error("Can't load %s repo: %s.", self.path, err)
-                self._remove()
+                await self._remove()
```

Comment:
There's another call to `_remove` on line 89.

Reply:
Good catch!